src/python/WMCore/ReqMgr/DataStructs/Request.py

```diff
@@ -20,6 +20,9 @@
 from WMCore.ReqMgr.DataStructs.RequestStatus import REQUEST_START_STATE
 from WMCore.ReqMgr.DataStructs.RequestError import InvalidSpecParameterValue
 
+#TODO: this need to be update from DB instead of hard coding for more flexibility
+RESULT_MASK_FOR_DAS = ["RequestName", "InputDatasets", "PrepID", "Campaign"]
+
```

Comment:
I will start with minimum. Actually, I didn't get about those property should not be a list. 
If the prepID is different for each Task/Step, how would you organize?